.circleci/config.yml

```diff
@@ -77,6 +77,7 @@ jobs:
                       - v0.3-torch_and_tf-{{ checksum "setup.py" }}
                       - v0.3-{{ checksum "setup.py" }}
             - run: pip install --upgrade pip
+            - run: pip install git+https://github.com/huggingface/nlp
```

Comment:
For now, using nlp master branch to test the integration with Trainer (we need some unreleased features). Not sure if we want to keep it this way for the foreseeable future or use the last release later on.